packages/config-cli/src/cli/bmc/config/config.update.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  BaseConfig,
+  ConfigDynamoBase,
+  Config,
+  ConfigTileSet,
+  ConfigImagery,
+  ConfigProvider,
+  ConfigVectorStyle,
+} from '@basemaps/config';
+import { BasemapsConfigObject } from '@basemaps/config/build/base.config.js';
+import { LogConfig, LogType } from '@basemaps/shared';
+import { ConfigDiff } from './config.diff.js';
+import PLimit from 'p-limit';
+
+export const Q = PLimit(10);
+
+export class Updater<S extends BaseConfig = BaseConfig> {
+  config: S;
+  prefix: string;
+  isCommit: boolean;
+  logger: LogType;
+
+  /**
+   * Class to apply an TileSetConfig source to the tile metadata db
+   * @param config a string or TileSetConfig to use
+   */
+  constructor(config: S, isCommit: boolean) {
+    this.config = config;
+    this.prefix = config.id.slice(0, 2);
```

Comment:
I think there is a `Config.getPrefix(config.id)` im not sure we have set that the prefix is only 2 chars.

Reply:
All updated. Thanks.